mitodl/open-discussions
# Fix courses with no course runs

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2203 

#### What's this PR do?
This makes `NewCourseWidget` safe against courses with no course runs

#### How should this be manually tested?
- Delete all course runs for a course that is showing up in the new courses widget on the home page.
- Confirm in master that this causes an error
- Confirm in this branch the homepage loads, but the course is no longer listed
